zanata-client-commands/src/main/java/org/zanata/client/commands/pull/PullCommand.java

```diff
@@ -198,13 +207,19 @@ public void run() throws Exception {
                     + "': existing source-language files may be overwritten/deleted");
             confirmWithUser("This will overwrite/delete any existing documents and translations in the above directories.\n");
         } else {
-            confirmWithUser("This will overwrite/delete any existing translations in the above directory.\n");
+            confirmWithUser(
+                    "This will overwrite/delete any existing translations in the above directory.\n");
         }
 
         if (getOpts().getPurgeCache()) {
             eTagCache.clear();
         }
 
+        Map<String, Map<LocaleId, TranslatedPercent>> statsMap = null;
```

Comment:
Would'n hurt to mark this `@Nullable`.  Also, please add a comment explaining the type params.